section2/exercises/if_statements.rb

```diff
@@ -61,5 +67,11 @@
 # Experiment with manipulating the value held within both variables
 # to make sure all above conditions output what you expect.
 
-cups_of_flour = 1
+cups_of_flour = 3
 has_sauce = true
+
+if cups_of_flour >= 2 && has_sauce == true
```

Comment:
Nice solution! Just FYI, you can omit the `== true`  since `has_sauce` is a boolean